test/ui/helper/Validators.test.js

```diff
@@ -0,0 +1,65 @@
+import { Validate, Validators } from "../../../ui/helper/Validators";
+
+describe("String", () => {
+    it("should validate the min length and not only spaces", () => {
+        const stringShort = "too short";
+        const stringLong = "this is not too short";
+
+        expect(Validate([Validators.String.minLength(10), stringShort])).toEqual("error");
+        expect(Validate([Validators.String.minLength(10), " ".repeat(20)])).toEqual("error");
+        expect(Validate([Validators.String.minLength(10), stringLong])).toEqual("success");
+    });
+
+    it("should validate the max length and not only spaces", () => {
+        const stringShort = "not long";
+        const stringLong = "this is too long!!";
+
+        expect(Validate([Validators.String.maxLength(10), stringLong])).toEqual("error");
+        expect(Validate([Validators.String.maxLength(10), "".repeat(3)])).toEqual("error");
+        expect(Validate([Validators.String.maxLength(10), stringShort])).toEqual("success");
+    });
+
+    it("should validate a non blank string", () => {
+        expect(Validate([Validators.String.nonBlank, null])).toEqual("error");
+        expect(Validate([Validators.String.nonBlank, undefined])).toEqual("error");
+        expect(Validate([Validators.String.nonBlank, NaN])).toEqual("error");
+        expect(Validate([Validators.String.nonBlank, ""])).toEqual("error");
+        expect(Validate([Validators.String.nonBlank, "   "])).toEqual("error");
+    });
+});
+
+describe("Number", () => {
+    it("should validate a natural number", () => {
+        expect(Validate([Validators.Number.natural, "4"])).toEqual("success");
```

Comment:
Don't remove valid cases such as actual numbers `4`, `-1`, etc. Plus, you're validating a `String`, not a `Number` here, it's a little bit misleading.